fast/stages/0-bootstrap/outputs.tf

```diff
@@ -112,6 +112,7 @@ locals {
     }
     custom_roles = module.organization.custom_role_id
     logging = {
+      log_sinks         = var.log_sinks
```

Comment:
should we pass this on via globals output file instead? it's a static quantity, I'm a bit wary of breaking the model and adding it to the stage outputs

Reply:
ok, just read the PR description snd it would not make sense :)

but I don't understand the use case, the variable type in stage 1 is this

```hcl
variable "logging" {
  # tfdoc:variable:source 1-tenant-factory
  description = "Logging configuration for tenants."
  type = object({
    project_id = string
    log_sinks = optional(map(object({
      filter = string
      type   = string
    })), {})
  })
  nullable = false
}
```

and everything is optional, how can it cause issues in TFE?

Reply:
I am sure there was a good reason, but I am struggling to remember why we create tenant log sinks in resman. I would try and revisit that (so we can remove log_sinks from the tenants variable) before adding sinks for non-tenant use.